subgraph/src/public-lock.ts

```diff
@@ -314,3 +323,28 @@ export function handleLockMetadata(event: LockMetadataEvent): void {
     lock.save()
   }
 }
+/**
+ * Create Receipt object for subgraph for key 'purchase'/'extend'/'renewal'
+ * @param {String} keyID - key id
+ * @param {event} Object - Object event
+ * @return {void}
+ */
+export function handleKeyReceipt(
```

Comment:
Lets user another name like create Receipt. Handle* should be reserved for events